etc/helm/pachyderm/templates/pachd/postgres-secret.yaml

```diff
@@ -2,7 +2,7 @@
 SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
 SPDX-License-Identifier: Apache-2.0
 */ -}}
-{{- if and .Values.pachd.enabled  (not .Values.postgresql.enabled) -}}
+{{- if and .Values.pachd.enabled  (not .Values.postgresql.enabled) (not .Values.global.postgresqul.postgresqlExistingSecretName) -}}
```

Comment:
ah! I think it's also postgresql is mispelled in .Values.global.**postgresqul**.postgresqlExistingSecretName